handlers/httpcontrollers/controller.go

```diff
@@ -20,8 +20,11 @@ const APP_BASE_URL = "http://local.exampleapp.com:3000"
 const APP_CALLBACK_PATH = "/oauth-callback"
 const AUTHORIZE_URL = RSO_BASE_URI + "/authorize"
 const TOKEN_URL = RSO_BASE_URI + "/token"
+const LOGOUT_URL = RSO_BASE_URI + "/logout"
 const APP_CALLBACK_URL = APP_BASE_URL + APP_CALLBACK_PATH
 
+var tokenResponse *TokenResponse
```

Comment:
I think a bit too much login-related business logic is starting to bleed into the "controller". We probably want to split out a class that contains the login page HTML and stores the token into its own package, and the controller can simply route to that class's functions. Can be a separate PR.

Reply:
OK! I'll mark it as improvement. Thanks